test/src/dash-adapter.spec.js

```diff
@@ -213,6 +213,38 @@ describe('DashAdapter: load', () => {
       });
   });
 
+  it('should set streaming.lowLatencyMod config to false on vod by default', done => {
+    try {
+      dashInstance = DashAdapter.createAdapter(video, vodSource, config);
+      video.addEventListener(EventType.LOADED_DATA, () => {
+        dashInstance._shaka.getConfiguration().streaming.lowLatencyMode.should.equal(false);
+        done();
+      });
+
+      dashInstance.load().then(() => {
+        video.play();
+      });
+    } catch (e) {
+      done(e);
+    }
+  });
+
+  it('should set streaming.lowLatencyMod config to true on live by default', done => {
+    try {
+      dashInstance = DashAdapter.createAdapter(video, liveSource, config);
+      video.addEventListener(EventType.LOADED_DATA, () => {
+        dashInstance._shaka.getConfiguration().streaming.lowLatencyMode.should.equal(true);
+        done();
+      });
+
+      dashInstance.load().then(() => {
+        video.play();
+      });
+    } catch (e) {
+      done(e);
+    }
+  });
+
   it('should load successfully when given a valid video to play', done => {
```

Comment:
we should add a test (or update an existing test) to check the use cases of setting lowLatencyMode manually in config (VOD source + set to true in config = true, Live source + set to false in config = false)